tools/two-machine-perf.ps1

```diff
@@ -0,0 +1,63 @@
+param (
+    [Parameter(Mandatory = $false)]
+    [ValidateSet("Debug", "Release")]
+    [string]$Config = "Release",
+
+    [Parameter(Mandatory = $false)]
+    [ValidateSet("x64")]
+    [string]$Arch = "x64"
+)
+
+Set-StrictMode -Version 'Latest'
+$ErrorActionPreference = 'Stop'
+
+# Set up the connection to the peer over remote powershell.
+Write-Output "Connecting to netperf-peer..."
+$Session = New-PSSession -ComputerName "netperf-peer" -ConfigurationName PowerShell.7
```

Comment:
I'd rather avoid pwsh 7 dependency (so far we have none) but if it makes remoting or anything else a lot easier, OK.